### PR TITLE
update to make open source policy more comprehensive

### PR DESCRIPTION
**Pull request by jimbarker**

### CODE REVIEW

**Review by dan-hill2802 — COMMENTED** [review]
Why the change from ordered to unordered lists? I think it makes it harder to reference

**Comment by jimbarker** [comment]
> Why the change from ordered to unordered lists? I think it makes it harder to reference

I don't have a strong preference either way. However, with ordered lists one must either: (a) hard-code the list reference and ensure reference numbers don't change; or (b) live with the version control challenge of having external references pointing to the wrong list item when the document changes. Having unordered lists avoids these challenges. Happy to change if you have a strong opinion on the subject.

**Comment by jimbarker** [comment]
Updated.

File: documentation.md [hunk 1]
```diff
@@ -1,34 +1,54 @@
...
-## Standards
-1. Documentation should be published in Markdown language.
-1. Diagrams should be embedded as images, with a link to the image for ease of viewing.
-1. Documentation should be subject to peer review and changes generally treated like code changes.
-1. Diagrams should be in the form of an architectural pattern or reusable as one.
+# DataWorks Open Source Documentation Policy
+
+## Purpose
+
+This document sets out local policy, for the Department for Work and Pensions (DWP) DataWorks team, on what types of documentation should be made available in the DWP public github repository, and what types of information can and cannot be included in documents and designs that are made publicly available.
```
Comment (dan-hill2802) [inline documentation.md]: `available in the DWP public github repository` -> `available in DWP public GitHub repositories`

File: documentation.md [hunk 2]
```diff
@@ -1,34 +1,54 @@
...
+
+It is expected that, at some point, this document will be superceded by a DWP-wide open-source documentation policy which covers the whole of the DWP public github repository.
+
+
+## Scope
+
+This policy covers documentation and designs only. The policy on infrastructure and application code will be covered either by a separate document, or by a future revision of this document.
```
Comment (dan-hill2802) [inline documentation.md]: remove: `or by a future revision of this document` it will be a separate doc, these will be linked to from the README.md

**Review by dan-hill2802 — CHANGES_REQUESTED** [review]

File: documentation.md (hunk 1 above)
Comment (jimbarker) [reply documentation.md]: Changed

File: documentation.md (hunk 2 above)
Comment (jimbarker) [reply documentation.md]: Changed.

**Review by dan-hill2802 — DISMISSED** [review]

**Review by dan-hill2802 — APPROVED** [review]

**Pull request merged**